README.md

```diff
@@ -272,3 +272,11 @@ WebSocket SubProtocol header.
 If no appropriate auth manager is configured or can be automatically determined
 then the fallback is the NoopAuthManager, which does no external validation
 calls and simply accepts any provided token or credentials.
+
+## JMX AUTHORIZATION HANDLING
+
+JMX connections into `container-jfr` are secured using a username and password 
+(unless the following flag is set: `"-Dcom.sun.management.jmxremote.authenticate=false"`).
```

Comment:
I don't think we should document this JVM flag specifically - it's what happens under the covers, and is really the only way that JMX auth can be enabled, but this might be confusing to an end user trying to figure out how to configure auth. In practice, all that matters are the env vars `CONTAINER_JFR_RJMX_{AUTH,USER,PASS}`. When actually deploying a ContainerJFR image somewhere, those env vars are how the user should expect to configure their auth, not the JVM flag directly, since configuring the JVM flags would require them to build their own custom image with a modified `entrypoint.sh`.

Reply:
In fact, I would lean toward not documenting the ability to disable the JMX auth at all. That's only something that should be done in development or testing. I can't think of a valid reason that a production deployment would want JMX auth on the ContainerJFR instance itself disabled. Documenting that it can be disabled may make it seem like this is considered reasonably good practice or even encouraged.

So, the docs should convey the fact that JMX auth on the ContainerJFR instance itself is always enabled, and the `USER/PASS` vars can simply be used to customize the credentials used for this always-on auth.